platform-hub-api/app/services/audit_service.rb

```diff
@@ -0,0 +1,20 @@
+module AuditService
```

Comment:
Was debating this… it seems simple enough – just a wrapper around the model, without much mapping logic, and figured the integration test would cover enough bases?